Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -106,6 +109,12 @@ export default class FilterComponent extends Vue {
                 isEnabled: this.config.modules[EntryType.Note],
                 numEntries: this.noteCount,
             },
+            {
+                type: EntryType.Note,
```

Comment:
Should this be `type: EntryType.MedicationRequest`?

Reply:
+1

Reply:
good catch!